### PR TITLE
Change font families

### PR DESCRIPTION
**Pull request by jngrad**
Use Linux Libertine instead of Latin Modern.

### CODE REVIEW

**Review by MakisH — COMMENTED** [review]
I don't see any issues, and this font actually seems to be more dense, condensing the whole paper slightly (around one page).

Here is a sample:

Screenshot from 2023-11-10 23-20-15

Compare this to the current, default font (CMU):

Screenshot from 2023-11-10 23-24-57

In the end, it's a question of style, and it probably only matters for the preprint.

**Comment by CaptainSifff** [comment]
So, the most recent version makes a difference of 25 pages vs 27 pages. To spell out the obvious, for this to work you need the right truetype Linux Libertine fonts installed, here(arch) it's called ttf-linux-libertine.
@jngrad A minor thing: In your Makefile, in the non-PDF/A branch I encountered this error: 
`Package pdfx Error: CreationDate is not properly supported;` 
a bit more over here: https://github.com/ndlib/nddiss/issues/4
But it could also have been me, having outdated build files...
To summarise:
I like the new look better.

**Comment by jngrad** [comment]
> @jngrad A minor thing: In your Makefile, in the non-PDF/A branch I encountered this error:
> `Package pdfx Error: CreationDate is not properly supported;`

Good catch! I'll fix that in a separate PR.
